script/latest_snapshot_version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#
+# Find the latest snapshot version for the specified branch.
+#
+# Example usage: ./latest_snapshot_version.py 7.x
+
+BRANCH=$1
+
+# The snapshots API has not yet been updated to use "main".
+if [ "$BRANCH" = "main" ]; then
+  BRANCH=master
+fi
+
+curl --silent "https://snapshots.elastic.co/latest/$BRANCH.json" | jq -r .version
```

Comment:
**issue**: `jq` doesn't return a proper exit code. We should use `-e` to set the exit code and try to handle empty input or missing key.

To reproduce:

```
~ % echo "" | jq -r .version
~ % echo $?
0
```

Fix attempt using `-e -n`:

```
~ % echo "" | jq -e -n -r 'input.version'
jq: error (at <stdin>:1): break
~ % echo $?
1
```